kiwi/storage/integrity_device.py

```diff
@@ -136,10 +160,19 @@ def create_integrity_metadata(self) -> None:
             header_string = '{0} {1} {2} integrity'.format(
                 metadata_format_version, filesystem, filesystem_mode
             )
-
-            parameters = [
-                f'internal_hash:{self.integrity_algorithm}'
-            ] + list(integrity_superblock['flags'])
+            if self.key:
+                secrettype = \
+                    f'{self.key.secrettype}({defaults.INTEGRITY_ALGORITHM})'
+                keyplaceholder = \
+                    f':dm-integrity-{self.key.secrettype}-secret'
```

Comment:
This string is supposed to be configurable as a key description in the kernel keyring. It should be an input string and it does not always have to be formatted like this which was just an example.

Reply:
I started a followup conversation about this part below